### PR TITLE
Chore: README update

### PR DESCRIPTION
**Pull request by tmladek**
## Description
Adds `superface compile` into README, and updates typos + language fixes (commas, wording) in its description.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [x] I have read the **CONTRIBUTION_GUIDE** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tmladek** [comment]
Hm, I just noticed VS Code *upgraded* the README's Table of Contents without me noticing in c3eb67c 😬 

Is that okay, or should I revert the change?

File: README.md [hunk 1]
```diff
@@ -191,6 +192,41 @@ EXAMPLES
 
 _See code: [src/commands/create.ts](https://github.com/superfaceai/cli/tree/main/src/commands/create.ts)_
 
+## `superface compile`
```
Comment (Jakub-Vacek) [inline README.md:195]: Nice! But :/ 
CLI uses oclif that have custom cli to generate command descriptions so we can do this automatically.

File: src/commands/compile.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { detectSuperJson } from '../logic/install';
 
 export default class Compile extends Command {
   static description =
-    'Compiles locally linked files in super.json. When running without --profileId flags all locally linked files are compiled. When running with --prfileId single local profile source and its local maps are compiled. When running with profileId and providerName single local profile and its single local map are compiled.';
+    'Compiles locally linked maps and profiles in `super.json`. When running without `--profileId` flag, all locally linked files are compiled. When running with `--profileId`, a single local profile source file, and all its local maps are compiled. When running with `--profileId` and `--providerName`, a single local profile and a single local map are compiled.';
```
Comment (Jakub-Vacek) [inline src/commands/compile.ts:17]: This was really needed 👍👍

File: README.md (hunk 1 above)
Comment (tmladek) [reply README.md:195]: Oh!

So should I re-generate this, or is this fine as it is? :)

Comment (Jakub-Vacek) [reply README.md:195]: Compile command is hidden for now so generation won't add it's description to readme. But after I make compile visible your changes in code itself will be propagated to readme. So I think it is fine.

**Pull request merged**